test/db/cmd/cmd_zignature

```diff
@@ -1118,3 +1118,318 @@ EXPECT=<<EOF
+za fcn.100e2b0c g cc=19 nbbs=35 edges=52 ebbs=1 bbsum=448
+za fcn.100e2b0c o 0x100e2b0c
+za fcn.100e2b0c v b0 b1 s52 s48 r74 r81
+za fcn.100e2b0c t func.fcn.100e2b0c.args=6 func.fcn.100e2b0c.arg.0="int32_t,arg1" func.fcn.100e2b0c.arg.1="int32_t,arg8" func.fcn.100e2b0c.arg.2="int32_t,arg_0h" func.fcn.100e2b0c.arg.3="int32_t,arg_1h" func.fcn.100e2b0c.arg.4="int32_t,arg_30h" func.fcn.100e2b0c.arg.5="int32_t,arg_34h"
+za fcn.100e2b0c h 3b28d8f368cb414615bda7313fc4cdc24096498e7183cc05165607770be26928
+EOF
+RUN
+
+NAME=PowerPC function with jumpback
+FILE=bins/elf/busybox-powerpc
+CMDS=<<EOF
+s 0x10002d70
+af
+zaf
+z*
+EOF
+EXPECT=<<EOF
+zs *
+za fcn.10002d70 b 38a0000038c000004bfffed4:ffffffffffffffffff000000
+za fcn.10002d70 g cc=12 nbbs=19 edges=29 ebbs=1 bbsum=304
+za fcn.10002d70 o 0x10002d70
+za fcn.10002d70 v b-4096 s4148 s4108 s4144 s8
+za fcn.10002d70 t func.fcn.10002d70.args=4 func.fcn.10002d70.arg.0="int32_t,arg_8h" func.fcn.10002d70.arg.1="int32_t,arg_100ch" func.fcn.10002d70.arg.2="int32_t,arg_1030h" func.fcn.10002d70.arg.3="int32_t,arg_1034h"
+za fcn.10002d70 h b73e65dc846183808d8e385076f9bbcd0b1dcdaa5652254fb55b3b159462a507
+EOF
+RUN
+
+NAME=PowerPC r_sign_fcn_bytes bounds check
+FILE=bins/elf/busybox-powerpc
+CMDS=<<EOF
+s 0x1000016c
+af
+e zign.maxsz = 30
+zaf
+z* ~za fcn.1000016c b
+EOF
+EXPECT=<<EOF
+za fcn.1000016c b 3d2000007c0802a6392900009421fff02f89000090010014419e001c3c80:ffffffffffffffffffffffffffffffffffffffffffffffffff000000ffff
+EOF
+RUN
+
+NAME=ARM function with hole
+FILE=bins/elf/libmagic.so
+CMDS=<<EOF
+s 0x78ea
+af
+zaf
+z*
+EOF
+EXPECT=<<EOF
+zs *
+za sym._Unwind_VRS_Get b 10b5041c042913d8081c00f072fd05031103030001200ce00220002b09d10f2a07d88240a418029a61681160181c00e0022010bd:ffffffffffffffffffffffffffffffffffffffffffffffff000000000000000000000000000000000000000000000000ffffffff
+za sym._Unwind_VRS_Get g cc=2 nbbs=4 edges=4 ebbs=1 bbsum=28
+za sym._Unwind_VRS_Get o 0x000078ea
+za sym._Unwind_VRS_Get r sym.__gnu_thumb1_case_uqi
+za sym._Unwind_VRS_Get v r0 r1
+za sym._Unwind_VRS_Get t func.sym._Unwind_VRS_Get.args=2 func.sym._Unwind_VRS_Get.arg.0="int16_t,arg1" func.sym._Unwind_VRS_Get.arg.1="int16_t,arg2"
+za sym._Unwind_VRS_Get h 3132d0ebfa3b792867a4ffc36455e088bf3fcffc7627bd48c64bf901308463db
+EOF
+RUN
+
+NAME=ARM function with jumpback
+FILE=bins/elf/libverifyPass.so
+CMDS=<<EOF
+s 0x0000183c
+af
+zaf
+z*
+EOF
+EXPECT=<<EOF
+zs *
+za sym.__aeabi_unwind_cpp_pr0 b 0030a0e3d4feffea
+za sym.__aeabi_unwind_cpp_pr0 g cc=27 nbbs=50 edges=75 ebbs=1 bbsum=1176
+za sym.__aeabi_unwind_cpp_pr0 o 0x0000183c
+za sym.__aeabi_unwind_cpp_pr0 r sym._Unwind_VRS_Get sym._Unwind_VRS_Get sym._Unwind_VRS_Set sym._Unwind_VRS_Get sym._Unwind_VRS_Get sym._Unwind_VRS_Set sym.__gnu_unwind_execute sym._Unwind_VRS_Get sym._Unwind_VRS_Set sym._Unwind_VRS_Set
+za sym.__aeabi_unwind_cpp_pr0 v s-48 s-20 s-36 s-24 s-16 s-15 s-12 s-68 s-8 s-44 s-40 s-52 s-56 s-4 s-32 r1 r0
```

Comment:
This made master branch red recently, probably because of recent changes in analysis that changed something in the identification of variables. Nonetheless, I think there is something wrong.

![image](https://user-images.githubusercontent.com/562321/87925651-e4b65280-ca80-11ea-823c-26a085d01bb5.png)

All variables in the function have `sp + X`, but here all variables are `s - Y`. Also, even though it says `s-84`, I don't see any variable in the function identified as `sp+0x54` nor `sp-0x54`. There seems to be something wrong.